Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorGenerateSnapshot.cpp

```diff
@@ -124,12 +125,28 @@ worker::Entity CreateGlobalStateManagerEntity(const NameToEntityIdMap& Singleton
 
 } // ::
 
-bool SpatialGDKGenerateSnapshot(FString SavePath, UWorld* World)
+worker::Option<FString> SetupSnapshotGenerationPath(FString SavePath)
```

Comment:
Remove the need for the worker::option and have it return an empty string if it fails. Also make the argument const&

Reply:
I'd go a little further and switch this back to a boolean return for success, and make the path a FString& parameter.